### PR TITLE
Quo2: Account list card blur

### LINKED ISSUE #17272: Quo2: account list card blur
Designs: https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=26013-280991&mode=design&t=6rncGnhPoMxovwTD-4

### PR DESCRIPTION
**Pull request by OmarBasem**
fixes: https://github.com/status-im/status-mobile/issues/17272

This PR implements blur for the account list card component.

[Designs](https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=26013-280991&mode=design&t=6rncGnhPoMxovwTD-4)

### CODE REVIEW

File: src/quo2/components/list_items/account_list_card/view.cljs
Comment (smohamedjavid) [inline src/quo2/components/list_items/account_list_card/view.cljs]: We can move these styles to `style.cljs`.

**Review by smohamedjavid — APPROVED** [review]
🚀

File: src/quo2/components/list_items/account_list_card/style.cljs
Comment (J-Son89) [inline src/quo2/components/list_items/account_list_card/style.cljs]: consider making this a map 👍

**Review by J-Son89 — APPROVED** [review]

**Comment by OmarBasem** [comment]
@Francesca-G a design review :)

**Review by Francesca-G — APPROVED** [review]
Component looks good ✨

@OmarBasem I've noticed a bug, if I select the blur theme it automatically goes to dark theme (which is correct), but if I try to press the theme switch button at the top when I'm In this state it goes back to light theme with blur: true, which shouldn't happen

Screenshot 2023-09-13 alle 13 23 29

**Comment by J-Son89** [comment]
Imo it would be nicer if we could just disable the blur toggle in light mode rather than swapping the theme etc 
Cc @ilmotta, @mohsen-ghafouri 

also while all these new updates to preview screens are welcome and very appreciated! I think we would all benefit from having some quick discussion  among 1 or 2 devs to agree upon the new idea/approach. Although perhaps that already happened in this case 🤔

**Review by ilmotta — APPROVED** [review]

**Comment by ilmotta** [comment]
> Imo it would be nicer if we could just disable the blur toggle in light mode rather than swapping the theme etc Cc @ilmotta, @mohsen-ghafouri
> 
> also while all these new updates to preview screens are welcome and very appreciated! I think we would all benefit from having some quick discussion among 1 or 2 devs to agree upon the new idea/approach. Although perhaps that already happened in this case 🤔

I certainly agree with you @J-Son89.

A bit off-topic, but to each their preference, whatever we decide to do, my vote will probably go to the solution that keeps previews dumb and flat so that we can completely auto-generate 90%+ of them (esp. if we get Malli in).

**Comment by OmarBasem** [comment]
> Imo it would be nicer if we could just disable the blur toggle in light mode rather than swapping the theme etc

Sound good, we can do that

**Review by briansztamfater — APPROVED** [review]

**Comment by OmarBasem** [comment]
> Component looks good ✨
> 
> @OmarBasem I've noticed a bug, if I select the blur theme it automatically goes to dark theme (which is correct), but if I try to press the theme switch button at the top when I'm In this state it goes back to light theme with blur: true, which shouldn't happen
> 
> Screenshot 2023-09-13 alle 13 23 29

@Francesca-G thanks for spotting this!

**Comment by OmarBasem** [comment]
@churik e2e seems to be failing. Does it have to do anything with this PR?

**Comment by churik** [comment]
e2e are not relevant now because of https://github.com/status-im/status-mobile/pull/17196

**Comment by churik** [comment]
Merging of all PRs is blocked until we find a reason

**Comment by churik** [comment]
@OmarBasem ready to merge if this component is not used anywhere in the app

**Pull request merged**
